index.html

```diff
@@ -0,0 +1,249 @@
+      <div class="profession">
+        <p>CANOPY</p>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>Back End Dev</p>
+        </div>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>2015</p>
+        </div>
+      </div>
+      <h3 class="description">
+        A daily selection of privately personalized reads;no accounts or
+        sign-ups required
+      </h3>
+
+      <ul class="languages">
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JavaScript</li>
+      </ul>
+
+      <button class="btn see-project">See project</button>
+    </section>
+    <section class="work">
+      <img src="./image/work4.png" />
+      <h3 class="work-title">MUlti-Post Stories</h3>
+      <div class="profession">
+        <p>CANOPY</p>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>Back End Dev</p>
+        </div>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>2015</p>
+        </div>
+      </div>
+      <h3 class="description">
+        A daily selection of privately personalized reads;no accounts or
+        sign-ups required
+      </h3>
+
+      <ul class="languages">
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JavaScript</li>
+      </ul>
+
+      <button class="btn see-project">See project</button>
+    </section>
+    
+  </section>
+  <section class="bg">
+    <section class="about-my-self">
+    <div class="top">
+      <h1>About Myself</h1>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <div class="social-media">
+        <p class="connect">LET’S CONNECT</p>
+        <ul class="social-links">
+          <li>
+           <a href="https://github.com/abenisamuel11" target="_blank" rel="noopener noreferrer"><img
+               src="./image/Github.png" alt="github logo"></a>
+         </li>
+        <li>
+           <a href="https://www.linkedin.com/in/abenezer-samuel/" target="_blank" rel="noopener noreferrer"><img
+               src="./image/linkdin.png" alt="linkdin logo" /></a>
+         </li>
+         <li>
+           <a href="" target="_blank" rel="noopener noreferrer"><img src="./image/A.png" alt="angle list icon" /></a>
+         </li>
+         <li>
+           <a href="https://twitter.com/" target="_blank" rel="noopener noreferrer"><img src="./image/twitter.png"
+               alt="twitter.icon" /></a>
+         </li>
+ 
+         <li>
+           <a href="" target="_blank" rel="noopener noreferrer"><img src="./image/medium.png" alt="medium logo" /></a>
+         </li>
+        
+ 
+         
+       </ul>
+      </div>
+      <button class="btn Get-my-resume">Get my resume</button>
+    </div>
+
+    <div class="skill-list">
+      <div class="language">
+        <div class="heading">
```

Comment:
- [ ] please kindly increase the font weight of the about myself title and make it bolder like int the original version
- [ ] kindly align the languages and frameworks to the left like in the original design.
yours
![Screenshot (1209)](https://github.com/abenisamuel11/mobile-version-finish-new-season2/assets/65298048/4dbce61e-14ea-46e3-9f4f-b007fa398d5b)
Figma
![Screenshot (1208)](https://github.com/abenisamuel11/mobile-version-finish-new-season2/assets/65298048/61464b17-9d59-451b-bda0-d7e53c4e1fb8)